hypothesis-python/tests/nocover/test_build_signature.py

```diff
@@ -14,7 +14,7 @@
 # END HEADER
 
 from inspect import signature
-from typing import get_type_hints
+from typing import get_type_hints, Literal
```

Comment:
Let's use e.g. `List[int]` instead of `Literal`; the latter is [new in Python 3.8](https://docs.python.org/3/library/typing.html#typing.Literal) and that complicates our test code a bit.